backends/edk2-compat/edk2-svc-validate.c

```diff
@@ -429,6 +431,8 @@ static int validateSingularESL(size_t *bytesRead, const unsigned char *esl, size
 			prlog(PR_INFO, "\tHash: ");
 			printHex(cert, cert_size);
 		}
+	}else if (varName && !strcmp(varName, "sbat")) {
+			rc = SUCCESS;
```

Comment:
Since this is getting merged as a seperate branch, we can let the formatting go unchecked, but please add a print statement if `verbose` is specified, similar how the dbx is handled right above it.